mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -1626,3 +1711,27 @@ func testAccMongoDBAtlasClusterConfigGCPRegionName(
 }
 	`, projectID, name, regionName)
 }
+
+func testAccMongoDBAtlasClusterConfigRegions(
+	projectID, name, replications string) string {
+	return fmt.Sprintf(`
+	resource "mongodbatlas_cluster" "test" {
+		project_id                              = "%[1]s"
+		name                                    = "%[2]s"
+		disk_size_gb            = 400
+	  num_shards              = 3
+	  provider_backup_enabled = true
+	  cluster_type            = "GEOSHARDED"
+	  // Provider Settings "block"
+	  provider_name               = "AWS"
+	  provider_disk_iops          = 1200
+	  provider_instance_size_name = "M30"
+	  %[3]s
+
+		lifecycle {
+		# avoid cluster has been auto-scaled to different instance size
```

Comment:
[nit] Seem like this comment has a typo? `# avoid cluster has been auto-scaled to different instance size` -> `# avoid cluster has been auto-scaled to different instance size`?

Reply:
I don't get it, what typo? :thinking:

Reply:
I made a typo hahah, I think it should be `# avoid cluster been auto-scaled to different instance size`?